render-only-sample/rosumd/RosUmdResource.cpp

```diff
@@ -331,7 +338,7 @@ RosUmdResource::CalculateMemoryLayout(
                 assert(m_mipLevels == 1);
 
                 assert((m_hwFormat == RosHwFormat::X8888) || (m_hwFormat == RosHwFormat::D24S8));
-                UINT sizeTileBytes = 64 * 64 * 4;
```

Comment:
Hi, Marek:

    You have good understanding of the VC4 tile format. Thanks for the good comments in the code.

    I suggest that you expand the work a bit:
1. Please define the byte size of the 4K tile, 1K sub-tile and 64 bytes macro-tile.
2. Please feel free to calculate the tile pixel width and height from 4K tile size and pixel size; or prepare a table for them. I think it is better not to reuse the VC4_BINNING_TILE_PIXELS, for 8888/32bit pixel the tile width and height are 32 (smaller), you are already using 32 right before ConvertBitmapTo4kTileBlocks().
3. Please do the same for the pixel dimension of sub-tile and macro-tile when appropriate.

    For now we have to use VC4_BINNING_TILE_PIXELS of 64 for linear render target.

                                       Thanks, Indy, 2/9/2016